.github/workflows/ci.yml

```diff
@@ -160,6 +160,26 @@ jobs:
     - name: test
       run: cross test --all --features unstable --target ${{ matrix.target }}
 
+  check_wasm:
+    name: Check wasm targets
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Install nightly with wasm32-unknown-unknown
```

Comment:
probably should use stable for this, there are some current nightlies without wasm32 target support

Reply:
addressed in 5342950 by adding a toolchain matrix, but I'm very confused about what happened with the actions. It ran some netlify stuff instead of the file I provided

Reply:
thanks